app/assets/stylesheets/navigation/breadcrumbs.scss

```diff
@@ -0,0 +1,42 @@
+.sci--layout-navigation-breadcrumbs {
+  --max-breadcrumbs-link-width: 180px;
+  align-items: center;
+  column-gap: 14px;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 14px 24px 8px;
+
+  .breadcrumbs-link {
+    color: var(--sn-blue);
+    display: inline-block;
+    max-width: var(--max-breadcrumbs-link-width);
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+
+    &:last-child {
+      color: var(--sn-grey);
+    }
+  }
+
+  .delimiter {
+    padding-bottom: 4px;
```

Comment:
Why you need it?

Reply:
Yes. without it, it doesn't look like figma